README.md

```diff
@@ -1 +1,1146 @@
+- In addition, we have used `` @kubernetes:Ingress `` which is the external interface to access your service (with path `` /`` and hostname ``ballerina.guides.io``)
+
+- Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. It points to the service file that we developed above and it will create an executable binary out of that. 
+This will also create the corresponding docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
+  
+```
+   $ballerina build message_transformation
+
+   Compiling source
+       message_transformation.bal
+
+   Generating executable
+       ./target/message_transformation.balx
+           @kubernetes:Service                      - complete 1/1
+           @kubernetes:Ingress                      - complete 1/1
+           @kubernetes:Deployment                   - complete 1/1
+           @kubernetes:Docker                       - complete 3/3 
+
+           Run following command to deploy kubernetes artifacts: 
+           kubectl apply -f ./target/message_transformation/kubernetes/
+```
+
+- You can verify that the docker image that we specified in `` @kubernetes:Deployment `` is created, by using `` docker images ``. 
+- Also the Kubernetes artifacts related our service, will be generated in `` ./target/message_transformation/kubernetes``. 
+- Now you can create the Kubernetes deployment using:
+
+```
+   $kubectl apply -f ./target/message_transformation/kubernetes 
+
+```
+
+- You can verify Kubernetes deployment, service and ingress are running properly, by using following Kubernetes commands. 
+
+```
+   $kubectl get service
+   $kubectl get deploy
+   $kubectl get pods
+   $kubectl get ingress
+```
+
+- If everything is successfully deployed, you can invoke the service either via Node port or ingress. 
+
+Node Port:
+ 
+```
+   curl -v http://localhost:9090/contentfilter -d '{"id" : 105, "name" : "ballerinauser", "city" : "Colombo 03", "gender" : "male"}' -H "Content-Type:application/json" -X POST 
+```
+
+Ingress:
+
+Add `/etc/hosts` entry to match hostname. 
+
+``` 
+   127.0.0.1 ballerina.guides.io
+```
+
+Access the service 
+
+``` 
+    curl -v http://localhost:9090/contentfilter -d '{"id" : 105, "name" : "ballerinauser", "city" : "Colombo 03", "gender" : "male"}' -H "Content-Type:application/json" -X POST
+```
+## Observability 
+Ballerina is by default observable. Meaning you can easily observe your services, resources, etc. Refer to [how-to-observe-ballerina-code](https://ballerina.io/learn/how-to-observe-ballerina-code/) for more information.
+However, observability is disabled by default via configuration. Observability can be enabled by adding following configurations to `ballerina.conf` file and starting the ballerina service using it.
```

Comment:
Would you start the Ballerina service normally here or pass some parameters in the run command? The way this is worded it sounds like you are using the conf file to start the Ballerina service.

Reply:
Done